cv_next/app/hall/page.tsx

```diff
@@ -1,54 +1,81 @@
-              className="w450:grid-cols-1 w450:gap-7 grid w-full grid-cols-2 gap-10 md:grid-cols-3 xl:w-1/2"
-            >
+    <>
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.0/css/all.min.css"
+      />
+      <title>Hall of fame</title>
+      {/* stars background */}
+      <ul className="crowns absolute left-0 top-0 z-[-1] h-[100%] w-[100%] overflow-hidden">
+        {Array.from({ length: 15 }).map((_, index) => (
+          <li key={index}></li>
+        ))}
+      </ul>
+      <div className="relative mx-auto h-full w-[700px] max-w-full xl:w-[1400px]">
+        <ScrollToTop />
+        <div className="select-none p-5 text-center">
+          <h1 className="hall-title mb-5 inline-flex items-center pt-3 text-2xl font-bold text-gray-900 dark:text-white sm:text-2xl md:text-3xl lg:text-4xl xl:text-5xl">
+            HALL OF FAME
+          </h1>
+          <h3 className="pb8 font-bold lg:text-lg">
+            Awesome people who contributed to the project
+          </h3>
+          <h3 className="pb-8 text-base font-light text-muted-foreground lg:text-lg">
+            Want to also be in this list? Contact us in the community chat and
+            win eternal glory!
+          </h3>
+          <ul>
+            <div className="flex max-h-[80vh] w-full flex-row-reverse flex-wrap justify-center justify-center self-center overflow-y-auto overflow-x-hidden xl:flex">
+
               {contributors.map((contributor, index) => (
-                <a
+                // key = index for now, but its bad practice..
+                // https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/
+                <div
                   key={index}
-                  target="_blank"
-                  className="rounded-base mb-3 rounded-lg border-2 border-b border-gray-200 bg-white p-6 text-base dark:bg-theme-800"
-                  style={{
-                    borderColor: "#000",
-                    boxShadow: "4px 4px 0px 0px #000",
-                    transform: "translate(4px, 4px)",
-                  }}
+                  className="card relative relative m-[1vh] mb-[10px] ml-[1vh] mr-[1vh] mt-[1vh] flex h-[25vh] w-[25vh] flex-col justify-between rounded-[10px] rounded-lg bg-primary-foreground p-4 p-[1em] text-center text-center"
                 >
-                  <p
-                    className="mt-3 text-lg sm:text-xl"
-                    style={{ fontWeight: "700" }}
-                  >
-                    {contributor.name}
-                  </p>
-                  <p
-                    className="mt-1 text-sm sm:text-base"
-                    style={{ fontWeight: "500" }}
-                  >
+                  <div className="name-image relative flex w-full items-center overflow-visible">
+                    <Image
+                      src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png"
```

Comment:
We have a default profile pic already, why not use it? It can also already be used theme-contextually (which, btw, check that your styling match)

Reply:
current default pic only has circle icon, Im looking for a square one. 
I'll tuckle this issue on my next push. right now will use this placeholder